src/pages/index.js

```diff
@@ -107,12 +107,12 @@ const submitButtonChangeAvatar = document.querySelector(
 });
 
 let cardSection;
-let userId; */
+let userId;
 
 /* -------------------------------------------------------------------------- */
 /*                          Verify Delete Card Modal                          */
 /* -------------------------------------------------------------------------- */
-/* 
+
 const cardVerifyDelete = new PopupCardDeleteVerify(".card-delete-verify");
 cardVerifyDelete.setEventListeners();
 
```

Comment:
```js

    api
      .deleteCardRequest(cardId)
      .then(element.remove())
      .then(cardVerifyDelete.close());
```


In blocks `then`, `catch`, `finally` you need to pass only the **reference** to the function (not a function call) otherwise the function is called immediately without waiting for the response from the server.



```js

  cardVerifyDelete.setSubmitAction(() => {
    api
      .deleteCardRequest(cardId)
      .then(() => element.remove())
      .then(() => cardVerifyDelete.close())
      .catch(console.error);
  });

```

And don't forget about `catch`